compiler/daml-extension/syntaxes/daml.json

```diff
@@ -117,7 +117,7 @@
         },
         {
           "name": "keyword.control.daml",
-          "match": "\\b(case|commits|fails|daml|else|ensure|exercise|if|import|in|let|match|must|of|on|qualified|scenario|template|then|type|data|where|while|with|anytime|signatory|observer|returning|to|agreement|do|does|test|def|fetch|controller|can)\\b"
+          "match": "\\b(case|commits|fails|daml|else|ensure|exercise|if|import|in|let|match|must|of|on|qualified|scenario|script|template|then|type|data|where|while|with|anytime|signatory|observer|returning|to|agreement|do|does|test|def|fetch|controller|can)\\b"
```

Comment:
side note: some of these keywords are very outdated, e.g., commits, fails, match, must